src/fcitx-rime.c

```diff
@@ -364,6 +365,15 @@ INPUT_RETURN_VALUE FcitxRimeGetCandWord(void* arg, FcitxCandidateWord* candWord)
     return retVal;
 }
 
+boolean FcitxRimePaging(void* arg, boolean prev) {
+    FcitxRime *rime = (FcitxRime *)arg;
+    boolean result = RimeProcessKey(rime->session_id, prev ? FcitxKey_Page_Up : FcitxKey_Page_Down, 0);
```

Comment:
Seems ibus-rime uses RimeProcessKey, I assume that's somewhat acceptable solution.